server/app/views/applicant/ProgramIndexView.java

```diff
@@ -83,17 +87,33 @@ public Content render(
             .setCondOnStorageKey("session_just_ended")
             .setDuration(5000));
 
-    bundle.addMainContent(
-        topContent(request, messages, personalInfo),
-        mainContent(
-            request,
-            messages,
-            personalInfo,
-            applicationPrograms,
-            applicantId,
-            messages.lang().toLocale(),
-            bundle,
-            profile));
+    // TODO: When the program filtering flag is removed, we can remove this conditional statement.
```

Comment:
**suggestion** Can we write this with the issue number for removing the flag, as `TODO(#5555):`

Reply:
Done